Connor Fitzgerald/social3.cpp

```diff
@@ -0,0 +1,33 @@
+#include "social.hpp"
+#include <iostream>
+
+int main() {
+	network nw;
+	// add three users
+	nw.add_user(profile{"mario", "Mario"});
```

Comment:
should use the string literal operator since you are passing in a string. You can specify  `using namepsace string_literals`